### DISCUSSION TITLE
Dynamic import not working with turbo repo internal package

### DISCUSSION THREAD

**Discussion by jgimitola** [vercel/next.js#48655] [Help]
### Summary

Hi, I am having some issues with a project i am developing. I am using turborepo and Nextjs. In the monorepo i have an `app` for the Nextjs app and a `package` where the ui components are implemented.

![image](https://user-images.githubusercontent.com/47477274/233517125-d4c31320-cf08-4bc3-894c-3049bf0be42f.png)

The thing is that when i try to use dynamic with my internal package nothing happens, but if i create a component in the app project and use dynamic the new chunk is created.

![image](https://user-images.githubusercontent.com/47477274/233516873-09e48146-2a77-4bcd-b3ae-90d4e1c66cb9.png)

Third request from top to bottom shows the new chunk for Field (local component) but there is no new chunk for OrangeCtaButton (package component)

I don't know what I could be doing wrong. Any idea what could be happening?

**Comment by JDaoMothership** [comment]
@jgimitola how did you end up resolving this?

**Reply by jgimitola** [reply]
I couldn't resolve directly. As a workaround, I imported the component from the internal package normally, wrapped it in a local Next.js component, and then used dynamic with the local path. While this worked, it doesn't feel like the best solution.

I think that that bundling and transpiling the package without using the NextJS transpilePackages would work but I am not sure.
